RIOT-OS/RIOT
# drivers/at: fixed faulty event prio name

Fixed a small oversight if using the at_urc_isr_highest pseudomodule.

### Contribution description

Basically just fixes an oversight in the at driver where it was checking if the at_urc_isr_high module was defined, which doesn't exist - it should have been at_urc_isr_highest. 
Just below that line it then defines AT_EVENT_PRIO as EVENT_PRIO_HIGH which also should have been EVENT_PRIO_HIGHEST.

### Testing procedure

Just include the at_urc_isr_highest module anywhere in a build and notice how it no longer outputs an error ;)

### Issues/PRs references

-
